public/catalog.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "geoplatform-stac",
     "stac_version": "1.0.0",
-    "title": "GeoPlatform STAC Catalog",
+    "title": " ",

```

Comment:
@chris-macdermaid is it ok to remove the title from the GP STAC Catalog? My concern is other entities may want to pull this catalog into their own stac browser and missing a title may lead to confusion. 

My original comment was on the redundancy of "GeoPlatform STAC" titles in the UI, but not sure its that big of a deal:
![image](https://github.com/GeoPlatform/stac-browser/assets/3847952/d141c570-a98e-4b8b-afe1-ed86c40b27ea)

Reply:
I don't know if it's strictly needed. In the Radiant Earth version of the STAC Browser without our customizations this will show up without a title at the top level.

https://radiantearth.github.io/stac-browser/#/external/gp-sit-us-east-1-stac.s3.amazonaws.com/catalog.json?.language=en

![image](https://github.com/GeoPlatform/stac-browser/assets/64213093/ec5a0c21-a383-40ae-b69f-c3981570a808)

Here's one with the title in Radiant Earth

https://radiantearth.github.io/stac-browser/#/external/explorer.digitalearth.africa/stac/?.language=en

![image](https://github.com/GeoPlatform/stac-browser/assets/64213093/799ad626-caa4-4ae1-946c-69a35822007a)